python/mxnet/_ffi/_ctypes/types.py

```diff
@@ -32,14 +32,11 @@ class TypeCode(object):
     NULL = 4
     MXNET_TYPE = 5
     MXNET_CONTEXT = 6
-    ARRAY_HANDLE = 7
-    OBJECT_HANDLE = 8
-    MODULE_HANDLE = 9
-    FUNC_HANDLE = 10
-    STR = 11
-    BYTES = 12
-    NDARRAY_CONTAINER = 13
-    NDARRAYHANDLE = 14
+    OBJECT_HANDLE = 7
+    STR = 8
+    BYTES = 9
+    PYARG = 10
```

Comment:
what's this type code for?

Reply:
It designates a python argument to be returned. For example, if parameter `out` is not `None`, then `out` also serves as the return value. Note that we should not return the `NDArray*` specified by `out`, as it creates another `NDArrayBase` in frontend, and thus one `NDArray*` in backend gets destructed twice by two frontend `NDArrayBase`